### DISCUSSION TITLE
UnityEngine.Application.add_logMessageReceived 가 작동하지 않습니다.

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#647] [Issue Report (Legacy)]
- 해당 함수로 로그를 뽑으려고 했는데, 작동하지 않습니다.
- 테스트한 코드와 에러는 아래와 같습니다.
![image](https://user-images.githubusercontent.com/5895396/164404577-5183fc6f-4e36-43a3-8f69-d217dfe943c8.png)
![image](https://user-images.githubusercontent.com/5895396/164404592-155e2fdb-e97d-45d6-8f34-62c5df8bab63.png)
![image](https://user-images.githubusercontent.com/5895396/164404658-82fd62e2-0e84-4171-9926-a2dd9ef6e031.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 이슈 확인해 보겠습니다.

**Comment by yoonyoungz** [comment]
https://docs.unity3d.com/ScriptReference/Application-logMessageReceivedThreaded.html 에 보면 Application.logMessageReceived 혹은 Application.logMessageReceivedThreaded 함수 통해서 로그를 출력할 수 있습니다.
